src/services/EventService.js

```diff
@@ -20,26 +20,25 @@ apiClient.interceptors.request.use(function (config){
 const userLogin = (payload, callback) => {
     apiClient.post('/auth/login', payload, callback)
         .then((res) => {
-            localStorage.setItem(TOKEN_KEY, res.data);
-            callback(true);
+            localStorage.setItem(TOKEN_KEY, res.data.token);
+            callback(true, res.data.name);
         }).catch(error => {
             console.log(error);
             callback(false);
-    });
+        });
 }
 
 const userLogout = () => {
-    delete apiClient.config.headers.authorization;
```

Comment:
Gonna need an explanation of why we can delete this line

Reply:
apiClient is not persisting the authorization header. It's persisted in localStorage so removing from localStorage is the correct way to do it